pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -213,6 +213,28 @@
   curricula_title_high: "High School Curricula"
   curricula_desc_high: "Browse the compelling courses and activities that invite you and your high school students to dig deeper into computer science."
 
+  curricula_pages:
```

Comment:
Am I remembering wrong, or were these added in a separate PR?

Reply:
Oh that's just from not having staging pulled, I'll do that now thank you!